openssl/src/aes.rs

```diff
@@ -1,15 +1,59 @@
 //! Low level AES functionality
 //!
-//! The `symm` module should be used in preference to this module in most cases.
+//! Advanced Encryption Standard (AES) provides symmetric key cipher that
```

Comment:
This should probably mention that most people looking for symmetric encryption should be using the [`Crypter`](https://docs.rs/openssl/0.9.19/openssl/symm/struct.Crypter.html) type. This module maps to the lower-level and more obscure options that aren't covered by the Crypter interface.